packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,93 @@
+@import '~@atomium/scss-utils/index';
+
+.atom-modal {
+  &__close {
```

Comment:
suggestion: to avoid a long BEM scope, we could use the classes outside of `atom-modal` because all CSS of Web Components are scoped, like

```scss

atom-modal {
  ...
}

.button-close {
 ... 
}
```

and use BEM just in each scoped block